docs/dataset/object-detection.md

```diff
@@ -0,0 +1,366 @@
+# How to Test Object Detection Models
+
+This guide outlines how to format your object detection data on Kolena, and walks through the steps to easily
+test object detection models.
+
+## Overview of Object Detection
+
+<div class="grid" markdown>
+<div markdown>
+Object detection is a widely applied computer vision task. Object detection aims to locate and identify objects
+within an image or video, typically with bounding boxes.
+Object detection has a wide range of applications from self-driving cars, facial recognition,
+video surveillance, medical imaging, robotics, and more. Therefore, it is crucial to evaluate object detection
+models to understand their performance and applicability in any real-world situation.
+</div>
+<figure markdown>
+![COCO Multiclass OD](../assets/images/task-od-multiclass.png)
+<figcaption markdown>Vehicle detection on an [MS COCO](https://paperswithcode.com/dataset/coco) image</figcaption>
+</figure>
+</div>
+
+??? info "Examples of Object Detection Datasets"
+
+    Object detection models require large amounts of annotated data to learn about objects of interest.
+    Some commonly used
+    datasets are: [MS COCO](https://paperswithcode.com/dataset/coco),
+    [Pascal VOC](https://paperswithcode.com/dataset/pascal-voc),
+    [Open Images](https://paperswithcode.com/dataset/open-images-v7),
+    and [ImageNet](https://paperswithcode.com/dataset/imagenet).
+
+## Object Detection Model Evaluation
+
+It is important to understand some core performance metrics and plots that are typically seen when
+evaluating object detection models.
+
+### Object Detection Metrics
+
+Evaluation of object detection models requires ground truths and model inferences. The ground truths in an image are
+objects outlined by bounding boxes each labeled with a class, typically annotated by human labelers. The model
+inferences for an image are the labeled bounding boxes having confidence scores (the model's certainty of
+correctness), as if the model annotated the image.
+
+??? info "Evaluation Metrics for Object Detection Models: Precision, Recall, F1-Score, and AP"
```

Comment:
Does this need to be in a collapsible section?  It's a bit odd to me that I don't see any list of metrics when I come to the metrics section of the docs. I normally use these collapsible callouts when there are <optional> information that doesn't need to be but nice to have on the page. 
 
It's fine to keep some details in the callout for further learning but we should at least list the metrics in the main paragraph of the section. Right now the paragraph just looks incomplete as it doesn't really talk about the metrics.